Aplib.Core/Belief/Belief.cs

```diff
@@ -8,8 +8,7 @@ namespace Aplib.Core.Belief
     /// (i.e., some piece of information of the game state as perceived by an agent).
     /// </summary>
     /// <remarks>
-    /// It implements the <see cref="IBelief"/> interface.
```

Comment:
The fact that it implements the `IBelief` interface is already visible in the docs. Extra documentation is dangerous because it might fall out of sync with the code if overlooked.